src/NuGet.Services.EndToEnd/AutoCompleteResultTests.cs

```diff
@@ -23,6 +23,7 @@ public AutocompleteResultTests(PushedPackagesFixture pushedPackages, ITestOutput
             _logger = logger;
         }
 
+        //[GalleryTestTheory] Uncomment this line, and remove the Theory below once the test is fixed
```

Comment:
I think that the name suggests that it should be used for all the Gallery tests. Is this the intention?